docs/appendix/apis/components/arm.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Arm API"
+linkTitle: "Arm"
+weight: 20
+type: "docs"
+description: "The arm API allows you to give commands to your arm components with code instead of with the graphical interface of the Viam app"
```

Comment:
This... feels odd. It seems like it doesn't need to be compared to the GUI.
```suggestion
description: "The arm API allows you to programmatically control your arm components."
```
or something